src/storage/conversion/RdfToQuadConverter.ts

```diff
@@ -29,12 +29,10 @@ export class RdfToQuadConverter extends RepresentationConverter {
       contentType: representation.metadata.contentType as string,
       baseIRI,
     });
-    data.pipe(errorStream);
-    data.on('error', (error): boolean => errorStream.emit('error', new UnsupportedHttpError(error.message)));
 
     return {
       dataType: DATA_TYPE_QUAD,
-      data: errorStream,
+      data: pipeStreams(data, errorStream),
```

Comment:
To improve understanding and readability:

```typescript
    const rawQuads = rdfParser.parse(representation.data, {
      contentType: representation.metadata.contentType as string,
      baseIRI,
    });
    // Wrap the stream such that errors are transformed
    // (Node 10 requires both writableObjectMode and readableObjectMode)
    const data = new PassThrough({ writableObjectMode: true, readableObjectMode: true });
    pipeStreams(rawQuads, data);

    return {
      dataType: DATA_TYPE_QUAD,
      data,
      metadata,
    };
```

Why do we need `as string`?

Reply:
This is actually something from #84 . It's because `contentType` could also be undefined, but this is checked in the `canHandle` function.